### ISSUE TITLE
Make it so you can import TestCase from test_plus

### ISSUE THREAD

**Issue by epicserve** [revsys/django-test-plus#4]
Just a thought, what about making it so you can import ``TestCase`` from ``test_plus.__init__.py``?

This just seems like it might be little nicer 

```python
from test_plus import TestCase
```

than this:

```python
from test_plus.test import TestCase
```

I think adding something like the following in ``test_plus.__init__.py`` would do the trick:

```python
from .test import TestCase

__all__ = [
    'TestCase',
]
```

**Comment by frankwiles** [comment]
Yeah I debated that as I was releasing it actually.  I didn't to keep it more like ```from django.test import TestCase``` but I think you're right.  Especially since this is all about typing less! :smiley: I'll work that into the next version.

**Comment by epicserve** [comment]
You're logic on why you didn't make sense to me. However another way to think about it, is with ``django.test``, django is a web framework and not a Test helper library, so ``from django import TestCase`` isn't as clear. Whereas with ``from test_plus import TestCase`` test_plus is a Test helper library.

Anyhow, thank you for making this public. It's ironic too because I was thinking before I started the day today that I needed to focus on making my tests better and more DRY! :)
